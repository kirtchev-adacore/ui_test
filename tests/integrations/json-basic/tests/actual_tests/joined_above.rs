#![deny(unused_mut, unused_variables)]

fn main() {
    //~| unused_mut
    //~v unused_variables
    let mut x = 0u32;
}
