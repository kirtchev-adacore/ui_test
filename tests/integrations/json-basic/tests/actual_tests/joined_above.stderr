error: unused variable: `x`
 --> tests/actual_tests/joined_above.rs:6:13
  |
6 |     let mut x = 0u32;
  |             ^ help: if this is intentional, prefix it with an underscore: `_x`
  |
note: the lint level is defined here
 --> tests/actual_tests/joined_above.rs:1:21
  |
1 | #![deny(unused_mut, unused_variables)]
  |                     ^^^^^^^^^^^^^^^^

error: variable does not need to be mutable
 --> tests/actual_tests/joined_above.rs:6:9
  |
6 |     let mut x = 0u32;
  |         ----^
  |         |
  |         help: remove this `mut`
  |
note: the lint level is defined here
 --> tests/actual_tests/joined_above.rs:1:9
  |
1 | #![deny(unused_mut, unused_variables)]
  |         ^^^^^^^^^^

error: aborting due to 2 previous errors

